frontend/src/scenes/experiments/ExperimentView/SummaryTable.tsx

```diff
@@ -181,8 +181,8 @@ export function SummaryTable(): JSX.Element {
                 ) as TrendExperimentVariant
                 const controlMean = controlVariant.count / controlVariant.absolute_exposure
 
-                const lowerBound = (credibleInterval[0] - controlMean) * 100
-                const upperBound = (credibleInterval[1] - controlMean) * 100
+                const lowerBound = ((credibleInterval[0] - controlMean) / controlMean) * 100
```

Comment:
I don't think I get this calculation, a comment would be helpful 🤔

Reply:
will add, thanks!